src/main/java/org/jabref/gui/customentrytypes/CustomizeEntryTypeDialog.fxml

```diff
@@ -55,19 +55,21 @@
                         </HBox>
                     </children>
                 </VBox>
-                <VBox prefHeight="400.0" prefWidth="100.0" spacing="10.0"
+                <VBox prefHeight="400.0" prefWidth="400.0" spacing="10.0"
                       HBox.hgrow="ALWAYS">
                     <children>
                         <Label text="%Required and optional fields"/>
                         <TableView fx:id="fields" minWidth="-Infinity"
                                    VBox.vgrow="ALWAYS">
                             <columns>
                                 <TableColumn fx:id="fieldNameColumn"
-                                             minWidth="150.0" prefWidth="-1.0" text="%Field"/>
+                                             minWidth="120.0" prefWidth="-1.0" text="%Field"/>
                                 <TableColumn fx:id="fieldTypeColumn"
-                                             minWidth="100.0" prefWidth="100.0" text="%Field type"/>
+                                             minWidth="130.0" prefWidth="150.0" text="%Field type"/>
                                 <TableColumn fx:id="fieldTypeActionColumn"
                                              maxWidth="40.0" minWidth="40.0" resizable="false"/>
+                                <TableColumn fx:id="fieldTypeMultilineColumn" text="%Multiline"
+                                             prefWidth="60.0" minWidth="50.0" resizable="true"/>
```

Comment:
Not so happy with the arrangement of the checkboxes and buttons.
The remove button should be the most right aligned button. the checkbox-column is too wide. Maybe rotate the label of the TableColumn 90 degrees to display it vertically? (https://stackoverflow.com/questions/27666087/vertical-column-text-in-tableview)

Reply:
I found a different solution for vertical alignment https://stackoverflow.com/a/74863464
In general this looks good, however, there seems to be some padding or margin missing.. Unfortunately I could not get scenic view to run on mac :(
<img width="438" alt="grafik" src="https://user-images.githubusercontent.com/320228/209942750-47ae277c-c3a3-47b9-8ecb-f40998990fd1.png">

Reply:
I will take a look

Reply:
can you commit your changes?

Reply:
Pushed to the wrong repo^^

Reply:
Fixed it  now 
<img width="699" alt="grafik" src="https://user-images.githubusercontent.com/320228/209968681-866c3d5e-7f47-4cc1-a85b-b70a548d3761.png">

Reply:
I'll look into it tomorrow. There are some UI tweaks I'm thinking of. 🤔